SmartDeviceLink/SDLTCPTransport.m

```diff
@@ -99,40 +98,12 @@ - (void)disconnect {
         return;
     }
 
-    // Attempt to cancel the `ioThread`. Once the thread realizes it has been cancelled, the thread will cleanup the input/output streams.
-    [self sdl_cancelIOThread];
-
-    if (self.ioThread == nil) {
-        SDLLogV(@"TCP transport successfully disconnected");
-        return;
-    }
-
-    // The `ioThread` could not be woken up to be cancelled. Switch to the `ioThread` and perform the cleanup of the input/output streams
-    [self performSelector:@selector(sdl_disconnect) onThread:self.ioThread withObject:nil waitUntilDone:NO];
-}
-
-/// Closes the the input/output streams on the `ioThread`
-- (void)sdl_disconnect {
-    /// We must close the input/output streams from the same thread that owns the streams' run loop, otherwise if the streams are closed from another thread a random crash may occur.
-    NSAssert([NSThread.currentThread.name isEqualToString:TCPIOThreadName], @"%@ should only be called on the IO thread", NSStringFromSelector(_cmd));
-
-    SDLLogD(@"Disconnecting TCP transport");
-
-    if (self.ioThread != nil) {
-        [self sdl_teardownStream:self.inputStream];
-        [self sdl_teardownStream:self.outputStream];
-
-        [self.connectionTimer invalidate];
-    }
-
-    self.ioThread = nil;
-
-    self.inputStream = nil;
-    self.outputStream = nil;
-
     [self.sendDataQueue removeAllObjects];
```

Comment:
This at least needs a debug log saying that it is disconnecting. (Same for `connect` I think, needs a "starting connection process" or "attempting to connect" log)

Reply:
Fixed.